sql_generators/glean_usage/templates/event_monitoring_aggregates_v1.query.sql

```diff
@@ -0,0 +1,109 @@
+-- Generated via ./bqetl generate glean_usage
```

Comment:
I made some improvements so that we need to deploy fewer artifacts. There is now one query that aggregates the daily event data for all applications (before we had a query for each app).